pipeline/pipeline.go

```diff
@@ -362,9 +403,8 @@ func (p *Pipeline) finalize(event *Event, notifyInput bool, backEvent bool) {
 
 	if notifyInput {
 		p.input.Commit(event)
-
-		p.totalCommitted.Inc()
-		p.totalSize.Add(int64(event.Size))
+		p.outputEvents.Inc()
+		p.outputSize.Add(int64(event.Size))
 
 		if len(p.outSample) == 0 && rand.Int()&1 == 1 {
```

Comment:
I was sure that I commited it, but probably some PR wasn't merged.
Let's replace `if len(p.outSample) == 0 && rand.Int()&1 == 1` with `if len(p.outSample) == 0` to print always not in 50% of time. @vano144 are you ok with that?

Reply:
It is only for debug  purpose. Why do you want change it? It is not about printing, it is about saving to outSample.

Reply:
I'm not changing this then

Reply:
@vano144 it now printed only in 50% of times because `rand.Int()&1 == 1`. @vitkovskii proposed remove this.